### DISCUSSION TITLE
[灌水]大新闻!珀尔曼跑了!

### DISCUSSION THREAD

**Discussion by Zzz-walnut** [share121/inter-knot#31] [其他]
<img width="266" alt="微信图片_20240809112542" src="https://github.com/user-attachments/assets/33c1b717-2f47-412b-8f0a-87df93ff4fe7">
天啊大家看新闻了吗，珀尔曼在远景案的开庭路上逃跑了!
新闻说飞艇的信号飞到外环之后消失了，治安局已经发通缉令了!

**Comment by shengwangyonghu12138** [comment]
唉，又要引起一阵骚动了

**Comment by MouxiuSama** [comment]
这个草芥人命的**，怎么给让他跑了，治安局的人都是吃白饭的吗？？

**Comment by DeepslateQAQ** [comment]
绳网团队提醒您，绳网并非法外之地，请勿对其他绳网用户发表骚扰、人身攻击等可能对他人造成困扰的内容。如您继续违反《绳网友好社区公约》，绳网团队可能会对您的账号作出禁言、封禁等处罚措施。

**Comment by MouxiuSama** [comment]
三楼QAQ我错了QAQ我再也不敢骂治安局了

**Comment by DeepslateQAQ** [comment]
（不会真有傻子当真了吧

**Comment by MouxiuSama** [comment]
不过是整活向罢了，我堂堂绳匠会怕治安局的无形大手？我可是要成为传奇绳匠的人
